server/controllers/match/placementRequests/occupancyViewController.ts

```diff
@@ -224,8 +219,9 @@ export default class {
     return async (req: ViewDayRequest, res: Response) => {
       const { token } = req.user
       const { id, premisesId, date } = req.params
-      const { criteria = [], excludeSpaceBookingId } = req.query
+      const { characteristics = [], criteria = [], excludeSpaceBookingId } = req.query
```

Comment:
I don't see any link to this controller that specifies `characteristics` as part of the query, where is this set? And how is it different from the existing `criteria`?

Reply:
Ah, debris from having a filter on the page - will remove